### DISCUSSION TITLE
[Script request] Kometa

### DISCUSSION THREAD

**Discussion by GameEnder** [community-scripts/ProxmoxVE#88] [Request script] [answered]
### Application Name

Kometa

### Website

https://kometa.wiki/

### Description

Kometa is a tool that lets you manage your media for Plex. It lets you change details about your Movies and TV Shows, like titles and descriptions. You can also organize your media into groups and add special effects to them.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Moorsy-AU** [comment]
This would be a great addition to the arr* scripts. Id also like Posterizarr included in the same script as the two work hand in hand to create a very beautiful thing. 
https://github.com/fscorrupt/Posterizarr
https://github.com/kometa-team/Kometa

**Comment by tremor021** [comment]
This app requires extensive manual configuration. LXC is easy to make, but you have a ton of options to configure before it becomes really usable.

**Comment by tremor021** [comment, marked as answer]
Will be available after #2281 merges

**Comment by Nelluk** [comment]
Some additional guidance for the user would be a good idea. There is no indication anywhere that Kometa has no web UI and requires editing the .yml files to do anything. When the script finishes it gives a web link to the newly-created LXC... but there is no web interface because Kometa is not a web app. It doesn't inform the user of where to look for the config.yml file.

IMO this script is just going to result in some people with no idea how to use Kometa and frustrated about it.

**Comment by tremor021** [comment]
Or you can go to their website and see what you're actually about to install and use? Or  you click on this handy little button:
![image](https://github.com/user-attachments/assets/ee2f4e49-9aa0-456d-ac2a-7202ef6667d6)
Or you go to their github and complain about not having UI

Plenty of options, none of them have anything to do with our install script.

**Comment by Nelluk** [comment]
I think the script exiting instructing the user to visit a website that does not exist is an issue. A single line of output along the lines of "Further required configuration must be done by editing /opt/kometa/config/config.yml" would go a long way to reducing user confusion.
